src/osgOcean/OceanScene.cpp

```diff
@@ -21,6 +21,10 @@
 
 #include <osg/Depth>
 
+
+#define SCREEN_WIDTH 1920

```

Comment:
In this code there are a lot of hard coded parameters, included these image render size. I only changed to fix underwater image effects. To make it a parameter i will need to study Osg Ocean, and now I do not have much time do it.